test/utils/fizzy_engine.cpp

```diff
@@ -135,8 +135,8 @@ WasmEngine::Result FizzyEngine::execute(
 {
     static_assert(sizeof(uint64_t) == sizeof(Value));
     const auto first_arg = reinterpret_cast<const Value*>(args.data());
-    const auto status =
-        fizzy::execute(*m_instance, static_cast<uint32_t>(func_ref), {first_arg, args.size()});
+    const auto status = fizzy::execute(
+        *m_instance, static_cast<uint32_t>(func_ref), span<const Value>(first_arg, args.size()));
```

Comment:
Is this change required?

Reply:
Yes. Otherwise compiler thinks these are two values: `first_arg` -> `bool` -> `Value` and `args.size()` -> `size_t` -> `Value`.

Reply:
I see. Fo first_arg it's probably pointer to integer conversion, rather than `bool`, I think `bool` -> `Value` doesn't work anymore, you deleted the tests for it.

Reply:
Without the change it resolves overloading to `Value(bool)` what gives the compilation error (as this constructor is deleted).